inc/commonitilobject.class.php

```diff
@@ -8177,33 +8177,27 @@ public static function getItemsTable() {
     *
     * @return array
     */
-   public function getLinkedItems(bool $addNames = true) :array {
```

Comment:
Should be removed from phpdoc

Reply:
CommonDBTM already has a docblock, then I removed it completely.